deployment/common/changeset/example/operations/deploy_and_mint_example_test.go

```diff
@@ -7,28 +7,22 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	chain_selectors "github.com/smartcontractkit/chain-selectors"
 	"github.com/stretchr/testify/require"
-	"go.uber.org/zap/zapcore"
 
-	cldf_chain "github.com/smartcontractkit/chainlink-deployments-framework/chain"
-
-	"github.com/smartcontractkit/chainlink/deployment/environment/memory"
-	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink-deployments-framework/engine/test/environment"
 )
 
 func TestDeployAndMintExampleChangeset(t *testing.T) {
-	lggr := logger.TestLogger(t)
-	e := memory.NewMemoryEnvironment(t, lggr, zapcore.InfoLevel, memory.MemoryEnvironmentConfig{
-		Chains: 1,
-	})
-	chain1 := e.BlockChains.ListChainSelectors(cldf_chain.WithFamily(chain_selectors.FamilyEVM))[0]
+	selector := chain_selectors.TEST_90000001.Selector
+	env, err := environment.New(t.Context(), environment.WithEVMSimulatedN(t, 1))
```

Comment:
should we be using `WithEVMSimulated` here?
```suggestion
	env, err := environment.New(t.Context(), environment.WithEVMSimulated(t, []uint64{selector}))
```

Reply:
Yeah this was a mistake. It just worked because the first n selector just so happens to match the one I chose explicitly